assets/scss/components/_calendar.scss

```diff
@@ -1 +1,55 @@
-@import "~flatpickr/dist/flatpickr"; 
+@import "~flatpickr/dist/flatpickr";
+
+/* meetup overrides */
+
+.flatpickr-calendar {
+	padding: $space-half;   
+	width: auto;
+
+	.svg--chevron-right svg {
+		fill: $C_textPrimary;
+	}
+
+	.svg--chevron-left svg {
+		fill: $C_separator;
+	}
+}
+
+span.flatpickr-weekday {
+	color: inherit;
+	font-weight: $W_bold;
+  }
+
+.flatpickr-current-month {
+	width:80%;
+	& span.cur-month {
+		font-weight: $W_normal;
+	}
+}
+
+.flatpickr-day.selected,
```

Comment:
This CSS is a little crazy. Is there a shorter/better way to select all of these things?

Reply:
I know. But I took this selector right from their stylesheet. I imagine its like this for a reason.